tokio/src/sync/mpsc/bounded.rs

```diff
@@ -157,6 +157,13 @@ impl<T> Receiver<T> {
 
     /// Blocking receive to call outside of asynchronous contexts
     ///
+    /// # Panics
+    ///
+    /// This function panics if the provided future panics, or if called
+    /// within an asynchronous execution context.
```

Comment:
The user doesn't provide any future here, so we shouldn't be talking about a "provided" future.

Reply:
Sorted :+1: